.github/workflows/release-validation.yaml

```
@@ -0,0 +1,19 @@
+name: release-validation
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  network-performance:
+    runs-on: ${{ github.repository_owner == 'coder' && 'depot-ubuntu-22.04-8' || 'ubuntu-latest' }}
```

```suggestion
    runs-on: ubuntu-latest
```
As this action only triggers a repository dispatch event, it should not need an 8-core runner and would do well on the free `ubuntu-latest` runner.

I am assuming standard GitHub action linux runners are free to use on public repos i.e., `ubuntu-latest`
This [page](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) says:
> The use of these runners on public repositories is free and unlimited.